### DISCUSSION TITLE
[Feature Request] Update popperjs to latest major version

### DISCUSSION THREAD

**Discussion by adrianfish** [ing-bank/lion#751] [General] [answered]
Overlays are using a deprecated version of popperjs, namely the old popperjs rather than the new @popperjs/core.

**Comment by tlouisse** [comment]
I think it's a great idea to upgrade to v2 of Popper. 
However, since the full Popper api is exposed as our public api, it currently would mean a breaking change in our own extension layer :)
But in the future we want to do this.

Alternatively, a solution could be to make a backwards compatibility layer that maps Popper v1 api to Popper v2 api (allowing us to use Popper v2). But I'm not sure how feasible/how much work that is.

**Comment by MathieuPuech** [comment]
It should solve an issue referred in code https://github.com/ing-bank/lion/blob/eb02b27010b5e56e7b69b704dfe37b56a1ebecb1/packages/overlays/src/OverlayController.js#L407

**Comment by tlouisse** [comment]
Nice 👍 Maybe something we can look into at a later point. 
Or maybe someone wants to do a contrubution that involves a compatibility map from v1 to v2? :)

**Comment by jorenbroekema** [comment]
We put it on our Lion v1 release tracker :) closing this for now as an enhancement, you can vote for it with 👍

**Comment by tlouisse** [comment, marked as answer]
Fixed here: https://github.com/ing-bank/lion/pull/1106
